platforms/python/prereqs/build.sh

```diff
@@ -47,6 +47,10 @@ if [ "$debianFlavor" == "stretch" ]; then
     echo "Hack flavor is: "$debianHackFlavor
 
     pythonSdkFileName=python-$PYTHON_VERSION.tar.gz
+
+    if [[ $PYTHON_VERSION == 3.6* ]]; then
+        PYTHON_GET_PIP_URL="https://bootstrap.pypa.io/pip/3.6/get-pip.py"
+    fi
     PYTHON_GET_PIP_URL="https://bootstrap.pypa.io/get-pip.py"
```

Comment:
Isn't the `PYTHON_GET_PIP_URL` value going to be overwritten by the next line?
Also, should we expect the `PIP_VERSION` to still be 20.2.3 in that case?

Reply:
yes, absolutely correct

Reply:
OK, thanks for fixing it!